terraform/admin/nomad/application-deployment/transit-app-example/transit-app-example.nomad

```diff
@@ -0,0 +1,76 @@
+job "transit-app-example" {
+    datacenters = ["us-east-1a", "us-east-1b", "us-east-1c"]
+    type = "service"
+    update {
+      stagger = "5s"
+      max_parallel = 1
+    }
+    group "transit-app-example" {
+        constraint {
+            operator = "distinct_hosts"
+            value = "true"
+        }
+        count = 1
+        task "transit-app-example" {
+
+            driver = "docker"
+            config {
+                image = "airedale/transit-demo:token"
+                volumes = ["local/config.ini:/usr/src/app/config/config.ini"]
+                network_mode = "host"
+                port_map {
+                    transitApp = 5000
+                }
+            }
+            template {
+                data = <<EOH
+                [DEFAULT]
+                LogLevel = WARN
+
+                [DATABASE]
+                Address=db.service.consul
+                Port=3306
+                User=vaultadmin
+                Password=vaultadminpassword
+                Database=my_app
+
+                [VAULT]
+                Enabled = True
+                DynamicDBCreds = False
+                ProtectRecords=False
+                Address=http://vault.service.consul:8200
+                Token=root
```

Comment:
don't have the source for this container, but is there a way to make this look more production like? source `VAULT_TOKEN` env variable?